sound/soc/intel/boards/sof_sdw_common.h

```diff
@@ -63,6 +63,8 @@ struct sof_sdw_codec_info {
 		     struct sof_sdw_codec_info *info,
 		     bool playback);
 
+	int (*exit)(struct device *dev);
```

Comment:
How about "cleanup" instead? It seems more commont to have either init/cleanup or init/cleanup pairs. Module exit is one exception in module.h, but module exit seems like the exception case.